.castor/docker.php

```diff
@@ -349,7 +349,7 @@ function create_default_context(): Context
         $data['power_shell'] = true;
     }
 
-    if ($data['user_id'] > 256000) {
+    if ($data['user_id'] > 256000 || false === $data['user_id']) {
```

Comment:
I would invert the condition to first check the false case.

Reply:
You're right, thank's ✅